.eslintrc.js

```diff
@@ -1,6 +1,6 @@
 module.exports = {
   parserOptions: {
-    ecmaVersion: 2018,
```

Comment:
Unsure whether this should be considered a "breaking change". If it is, a major release still seems like overkill.